agent/src/main/scala/za/co/absa/atum/agent/model/Measure.scala

```diff
@@ -35,11 +35,12 @@ sealed trait Measure {
 trait AtumMeasure extends Measure with MeasurementProcessor
 
 final case class UnknownMeasure(measureName: String, measuredColumns: Seq[String], resultValueType: ResultValueType)
-  extends Measure
+    extends Measure
 
 object AtumMeasure {
 
-  case class RecordCount private (measureName: String) extends AtumMeasure {
+  final case object RecordCount extends AtumMeasure {
+    val measureName: String = "count"
```

Comment:
To make it look like the other functions:
```suggestion
    val measureName: String = "count"
    
    def apply() = this
```
Maybe with some extra tweaking like declaring `final class RecordCount` and making final case object RecordCount extends RecordCount`.

Reply:
Also makes the change backward compatible wherever `RecordCount()` was used.